test/example/README.md

```diff
@@ -45,6 +45,10 @@ ok
 Also, it's interesting is to experiment with various variants of CTL-Z and
 CTL-C on both the server and the client.
 
+Furthermore, there is an almost identical variant called qt_ssl.erl that can
+be used to perform comaparisions between quic and SSL. Both speed and memory
+consumption tests can be performed. Spoiler, quic is both smaller and faster
+than OTP SSL.
```

Comment:
good to know!
We also did some benchmarks in early stage last year
https://www.emqx.com/en/blog/mqtt-over-quic

Some info is outdated, and we will update them after the major EMQX 5.0 major release.